ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -375,6 +376,24 @@ func (r *RayJobReconciler) getSubmitterTemplate(rayJobInstance *rayv1alpha1.RayJ
 		r.Log.Info("User-provided command is used", "command", submitterTemplate.Spec.Containers[0].Command)
 	}
 
+	// Set PYTHONUNBUFFERED=1 for real-time logging, unless user already set it.
+	userProvidedValue := ""
+	if common.EnvVarExists(PythonUnbufferedEnvVarName, submitterTemplate.Spec.Containers[0].Env) {
+		for _, envVar := range submitterTemplate.Spec.Containers[0].Env {
```

Comment:
It is a bit weird to iterate through the loop again after calling `EnvVarExists`.

Reply:
We may consider returning `bool, EnvVar`.